### PR TITLE
Add 古塘咖啡 & 星巴克（望京店）

### PR DESCRIPTION
**Pull request by beta**

### CODE REVIEW

**Comment by beta** [comment]
Here you go :)

**Comment by beta** [comment]
@xhacker Count updated :).

File: beijing.geojson [hunk 1]
```diff
@@ -1212,6 +1212,43 @@
...
+        "marker-symbol": "cafe"
+      },
+      "geometry": {
+        "type": "Point",
+        "coordinates": [
+          116.482,
+          40.001
+        ]
+      }
+    },
```
Comment (xhacker) [inline beijing.geojson]: Please make sure to check the CI result... Looks like you have an extra comma here.

**Comment by xhacker** [comment]
Also, it’s hard to merge because your branch has a lot of conflicts with master. When you want to do a contribution, the best way is to:

1. [Merge the upstream change to your fork](https://help.github.com/articles/merging-an-upstream-repository-into-your-fork/)
2. Open a new branch from master (`git checkout master; git checkout -b new-branch-name`)
3. Commit your change
4. Open a pull request

By doing this way, there usually won’t be a lot of conflicts.

**Comment by beta** [comment]
Sorry for the conflicts and I'll close this PR and open another one in a correct way.

**Pull request closed without merging**